gatearray.py

```diff
@@ -9,16 +9,17 @@
 from collections import namedtuple
 
 
-START = namedtuple('START', 'n')
-X = namedtuple('X', 'i')
 CNOT = namedtuple('CNOT', 'ctrl target')
 H = namedtuple('H', 'i')
 I = namedtuple('I', 'i')
-
+START = namedtuple('START', 'n')
+SWAP = namedtuple('SWAP', 'a b')
```

Comment:
We might want to start building a macro library instead of expanding the set of gates.

Reply:
Might be, but I'll merge this for now so we can write qcode with CNOT operations on non adjacent qubits.